### DISCUSSION TITLE
Kaspersky virus report

### DISCUSSION THREAD

**Discussion by rainsdm** [openssl/openssl#23750] [Q&A] [answered]
Please do NOT use issues to ask questions about OpenSSL.

Instead, please use the [Q&A category in Discussions](<https://github.com/openssl/openssl/discussions/new?category=q-a>)
to ask your question.

Kaspersky thinks these files are virus:
![Snipaste_2024-03-05_16-27-04](https://github.com/openssl/openssl/assets/59646708/c18b676f-27e1-44b0-8433-fb5d6025f5a0)

Please make sure if they are really safe.

The pictures below shows the files which were deleted:
![Files_that_have_deleted](https://github.com/openssl/openssl/assets/59646708/9b963ca7-a078-4663-ae58-fe7cd3f4a856)

**Comment by mattcaswell** [comment, marked as answer]
Your screenshot didn't actually show what specific files it was complaining about. But in any case, assuming that the files you have are identical to those in the OpenSSL repository, then this is a false positive from Kaspersky.

These files are deliberately malformed ASN.1 files used for testing purposes. The "DoS" in the name above suggests that Kaspersky thinks these are a denial-of-service attempt. Some malformed ASN.1 could indeed be a DoS attempt if used in a "live" environment. But these are test files and are safe.
